components/gitpod-db/src/typeorm/migration/1679391121672-AlignAdminUsers.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { MigrationInterface, QueryRunner } from "typeorm";
+import { BUILTIN_INSTLLATION_ADMIN_USER_ID } from "../../user-db";
+
+const OLD_BUILTIN_INSTLLATION_ADMIN_USER_ID = "builtin-installation-admin-user-0000";
+
+/**
+ * Because we changed IDs of the "admin-user" in this PR (https://github.com/gitpod-io/gitpod/pull/15974/files) we have to make sure we're all aligned on that new id.
+ */
+export class AlignAdminUsers1679391121672 implements MigrationInterface {
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        // Double add it to make sure every installation has one with the new ID
+        await queryRunner.query(
+            `INSERT IGNORE INTO d_b_user (id, creationDate, avatarUrl, name, fullName, rolesOrPermissions, blocked) VALUES ('${BUILTIN_INSTLLATION_ADMIN_USER_ID}', '${new Date().toISOString()}', '', 'admin-user', '', '["admin"]', TRUE)`,
```

Comment:
Can't we just drop any existing and recreate fresh with the static `BUILTIN_INSTLLATION_ADMIN_USER_ID`?

Reply:
I don't think that's needed; there are no changes to that user, except the one id.
IMO it does not matter if it's two migration or one. This way we even get better feedback on failure.